<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YDSync 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>290</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Язык</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_token">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>290</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Токен доступа Вашего Яндекс.Диска:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_local">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>290</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Локальная папка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_yddir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>290</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Папка на Яндекс.Диске:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_ignoreextension">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>290</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Игнорировать файлы с расширением:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_rus">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>25</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: rgb(153, 193, 241);
spacing: 8px; /* расстояние между индикатором и текстом */
padding: 4px;
}

QRadioButton::Indicator {
border: 2px solid white;
border-radius: 7px;
}

QRadioButton::indicator:checked {
border: 2px solid #007acc;
background-color: #007acc;
}

</string>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_eng">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>25</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
color: rgb(153, 193, 241);
spacing: 8px; /* расстояние между индикатором и текстом */
padding: 4px;
}

QRadioButton::Indicator {
border: 2px solid white;
border-radius: 7px;
}

QRadioButton::indicator:checked {
border: 2px solid #007acc;
background-color: #007acc;
}

</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_token">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>339</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color: rgb(153, 193, 241);
border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_local">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>339</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color: rgb(153, 193, 241);
border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_helptoten">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_local">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_yddir">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>339</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color: rgb(153, 193, 241);
border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ignoreextension">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>339</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color: rgb(153, 193, 241);
border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_local_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_ignorefiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>290</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Игнорировать файлы:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ignorefiles">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>339</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color: rgb(153, 193, 241);
border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_logsize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>290</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Размер Лог-файла не более:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_logsize">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color: rgb(153, 193, 241);
border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_kb">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>258</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>КБ</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_prompt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>609</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 10px;
color: rgb(143, 240, 164);
border-right: 1px solid white;
border-top:1px solid white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_local_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(38, 162, 105);</string>
    </property>
    <property name="text">
     <string>Начать синхронизацию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_local_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(165, 29, 45);</string>
    </property>
    <property name="text">
     <string>Остановить</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_time">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 10px;
color: rgb(143, 240, 164);
border-top: 1px solid white;</string>
    </property>
    <property name="text">
     <string>000:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_help">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>25</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Помощь</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_openlog">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Открыть лог-файл</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
